lua/cfc_ulx_commands/timed_punishments/server/data.lua

```diff
@@ -18,6 +18,18 @@ local function query( query, ... )
     return sql.Query( query )
 end
 
+local function formatPunishments( result )
+    local punishments = {}
+    if result == nil then return punishments end
+    if result == false then return ErrorNoHaltWithStack( "Query failed when retrieving punishments!" ) end
+
+    for _, p in ipairs( result ) do
+        punishments[p.punishment] = tonumber( p.expiration )
+    end
+
+    return punishments
+end
```

Comment:
Made a new `get` method on the Database that returns only currently-valid punishments, and another that returns all punishments (expired or otherwise), but they'll both be formatted the same way so this helper method handles both